libr/core/cmd_open.c

```diff
@@ -210,6 +210,7 @@ static void list_maps_visual(RIO *io, ut64 seek, ut64 len, int width, int use_co
 			for (j=0; j<width; j++) {
 				ut64 pos = min + (j*mul);
 				ut64 npos = min + ((j+1)*mul);
+				// TODO address space wraparound
 				if (s->from<npos && (s->to)>pos)
 					io->cb_printf ("#");
 				else io->cb_printf ("-");
```

Comment:
add braces

Reply:
It's already there and I do not have time to make it correct. I just left a comment there as a reminder. It will be changed, but not now.